terraform/modules/kubernetes/main.tf

```diff
@@ -18,7 +18,7 @@ resource "kubernetes_deployment" "webapp" {
       }
       spec {
         node_selector = {
-          "kubernetes.azure.com/agentpool" = "applications"
+          "kubernetes.azure.com/agentpool" = "apps2"
```

Comment:
Should we use the new tag here? CC @smneal

Reply:
The new tag is `"teacherservices.cloud/node_pool": "applications"` but it's only in the production cluster at the moment.

Reply:
https://github.com/DFE-Digital/teacher-services-cloud/pull/47